docs/host/host.md

```diff
@@ -23,6 +23,16 @@ Because Harvester is built on top of Kubernetes and uses etcd as its database, t
 
 For admin users, you can click **Enable Maintenance Mode** to evict all VMs from a node automatically. It will leverage the `VM live migration` feature to migrate all VMs to other nodes automatically. Note that at least two active nodes are required to use this feature.
 
+If you want to individually force a shutdown of a VM instead of migrating it to another node, add the label `harvesterhci.io/maintain-force-shutdown-strategy` and one of these values to those VMs:
+
+- `RestartOnEnable`: Restart the VM after the node has finally switched into maintenance mode. The VM will be scheduled on a different node.
```

Comment:
The `RestartOnEnable` says it will work: `after the node has finally switched into maintenance mode`,

What's the difference of this mode `RestartOnEnable` and the `live-migration`? Give use a chance to execute some scripts?

Reply:
> What's the difference of this mode `RestartOnEnable` and the `live-migration`? Give use a chance to execute some scripts?

Yes, that's the idea. 

The general idea with the options was to offer the user as many options as possible so that he can do what suits his scenario.